src/frontend/Komponenter/Behandling/Endring/EndringPersonopplysninger.tsx

```diff
@@ -114,6 +118,22 @@ const Personopplysningsendringer: React.FC = () => {
                             endringen er relevant må man oppdatere registeropplysninger og vurdere
                             aktuelle vilkår.
                         </BodyLongMedium>
+                        <Button
+                            size={'small'}
+                            variant={'primary'}
+                            loading={nyGrunnlagsdataHentes}
+                            onClick={() => {
+                                if (!nyGrunnlagsdataHentes) {
+                                    settNyGrunnlagsdataHentes(true);
+                                    oppdaterGrunnlagsdataOgHentVilkår(behandlingId).then(() => {
+                                        nullstillGrunnlagsendringer();
+                                        settNyGrunnlagsdataHentes(false);
```

Comment:
Dette bør ut i en egen funksjon? Her skjer det litt mye spennende inline for min del :)